website/docs/registry/faq.mdx

```diff
@@ -0,0 +1,110 @@
+---
+page_title: Terraform Registry FAQ
+description: Troubleshooting common Terraform Registry problems
+---
+
+## Common publishing issues
+
+### How can I fix a formatting issue in the documentation for my provider?
+
+If your provider's documentation is not showing up in the registry the way you would like it to, you can use the [documentation preview tool](https://registry.terraform.io/tools/doc-preview) to see how the Terraform Registry will format your documentation _before_ you commit the documentation to a new provider release. [More information about how to build provider documentation is available here](/terraform/registry/providers/docs).
+
+If you have already released a version of your provider and you are unhappy with the appearance of it's documentation, [we recommend you fix the documentation issue and include it in a new release](#how-can-I-correct-a-bug-typo-or-documentation-error-in-a-version-of-my-provider-module-or-policy-library-once-it-has-been-published).
```

Comment:
Should we explicitly say

"You cannot update your documentation without a new release." "we recommend" seems a bit like there are other options